@navikt/ds-react/grid/src/Cell.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { forwardRef, HTMLAttributes } from "react";
+import cl from "classnames";
+import "@navikt/ds-css/grid/index.css";
+
+type Column = 1 | 2 | 3 | 4 | 5 | 6 | 7 | 8 | 9 | 10 | 11 | 12;
+export interface CellProps extends HTMLAttributes<HTMLDivElement> {
+  children?: React.ReactNode;
+  className?: string;
+  white?: boolean;
+  padding?: boolean;
```

Comment:
Jeg ville gjettet på at en prop `padding` tok et tall eller en string, litt som style-object. Om vi skal ta en boolean for å sette padding foreslår jeg heller navnet `withPadding`.

Reply:
Godt forslag, kan endre til withPadding :)

Reply:
Denne burde kanskje være default={true} og må deaktiveres eksplisitt? 
Vi ønsker vel at så mange som mulig skal benytte lik padding :)

Reply:
Fjerner padding parameteret, ref kommentaren om padding var nødvendig